src/components/shared/Stats.tsx

```diff
@@ -1,13 +1,29 @@
 import '../../styles/stats.scss';
+
+type statsObject = {
+  [key: string]: number,
+}
+
 interface statsProps {
-  battery: number;
+  playerStats: statsObject,
 }
 
-export default function Stats(props:statsProps): JSX.Element {
+export default function Stats(props: statsProps): JSX.Element {
+  const { playerStats } = props;
+
+  const renderStats = () => {
```

Comment:
Since there isn’t much code and this is only used once, I’d prefer to in-line this